src/pages/UserList/ContextMenu/Modals/ResetTwoFAModal.js

```diff
@@ -11,32 +11,32 @@ import {
 import PropTypes from 'prop-types'
 import React, { useState } from 'react'
 import { useFetchAlert } from '../../../../hooks/useFetchAlert.js'
+import styles from './ResetTwoFAModal.module.css'
 
-const Disable2FaModal = ({ user, refetchUsers, onClose }) => {
+const ResetTwoFAModal = ({ user, onClose }) => {
     const engine = useDataEngine()
     const [loading, setLoading] = useState(false)
     const { showSuccess, showError } = useFetchAlert()
 
-    const handleDisable2Fa = async () => {
+    const handleReset2FA = async () => {
         setLoading(true)
         try {
             await engine.mutate({
                 resource: `users/${user.id}/twoFA/disabled`,
                 type: 'create',
             })
             const message = i18n.t(
-                'Disabled two factor authentication for "{{- name}}" successfuly',
+                'Two factor authentication for "{{- name}}" has reset successfully',
                 {
                     name: user.displayName,
                 }
             )
             showSuccess(message)
-            refetchUsers()
```

Comment:
I don't see why we would refetch users after resetting the 2FA for one user as this will not change the user information? I think this was relevant when we expected the users response would contain `twoFactorEnabled` property (e.g. we refetch users to refresh that property for the user we just reset 2FA for)